### ISSUE TITLE
Clarification on FID / Inception Score evaluation setup

### ISSUE THREAD

**Issue by Trivezzz** [jwmao1/MedSegFactory#13]
Hi; I’m trying to reproduce/interpret the reported FID and Inception Score (IS) results, and I’d appreciate a few clarifications about your evaluation protocol:
1) How many generated images are used to compute FID and IS (per dataset)?  
2) Do you compute these metrics against the training set images or the test set images?
3) Would it be possible for you to provide (or point to) the generated image sets you used to compute FID/IS?

Any details you can share would be extremely helpful.
Thanks in advance!

**Comment by jwmao1** [comment]
Hi @Trivezzz, thanks for your attention to our work. We compute these metrics on the [test set](https://huggingface.co/datasets/JiaMao/test). More details can be found in the paper. Thanks again for your issue.
